package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "hydraclient.js",
+    "version": "0.0.1",
+    "scripts": {
+        "test": "node node_modules/karma/bin/karma start",
+        "test-once": "node node_modules/karma/bin/karma start --single-run"
+    },
+    "dependencies": {
+        "jsonld": "^0.4.12",
+        "typescript": "^2.3.3",
+        "whatwg-fetch": "^2.0.3"
```

Comment:
I've got someone asking in "my" heracles to use [isomorphic-fetch](https://www.npmjs.com/package/isomorphic-fetch) so that the client can be used both in browser and Node. Let's do that from get-go.